### PR TITLE
Feature/sub events mutations

### PR DESCRIPTION
**Pull request by kevinhartmann23**
## *Purpose:*
*What does this merge do? Check all that apply.*
- [ ] Adds a new feature
- [ ] Improves an existing feature
- [x] Fixes a bug
- [ ] Cleans up (logs, tests, etc.)

## *Description: What does this PR do?*
*Briefly, but clearly describe what you did, and explain any new code.*
- Fix timezone in calendar
- 

## *Instructions for reviewer:*
*Where should the reviewer start?*
*How should this be manually tested?*
*Any background context you want to provide?*
*What are the relevant tickets?*

## *Issues:*
*Are there any concerns, issues, or bugs in this branch? If so describe them.*

## Checklist:
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have used tests to prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my change

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -8,26 +8,31 @@ import Form from './form/Form'
...
-        <Route path="/new-event" component={Form} />
-        <Route path="/contacts" component={Contacts} />
-        <Route path="/event-details" component={Event} />
-        {/* <Route path='/loading' component={Loading} /> */}
-        {/* <Route path="/event/:name" render={blah blah match what not here} /> */}
-        <Route component={NotFound} />
-      </Switch>
-      <NavBar/>
+      <Header />
+      {appError && <NotFound />}
```
Comment (richardltyler) [inline src/components/App.js:22]: Yeah good idea here Kevin. Nailed it!

**Review by kristenmb — APPROVED** [review]
Amazing work, you are a champion

File: src/components/calendar/Calendar.js [hunk 2]
```diff
@@ -44,6 +44,7 @@ const Calendar = () => {
   return (
        <CalendarWrapper>
          <FullCalendar
+            timeZone= 'UTC'
```
Comment (richardltyler) [inline src/components/calendar/Calendar.js:47]: Nice to know that we could change these!

File: src/components/form/Form.js [hunk 3]
```diff
@@ -28,7 +28,6 @@ const Form = ({location}) => {
       history.push({
         pathname: "/",
       })
-      console.log("offline bro")
```
Comment (richardltyler) [inline src/components/form/Form.js:31]: How will we know that we're offline now?

**Review by richardltyler — APPROVED** [review]
ITS THE FINAL COUNTDOOOOOOOOOWN

File: src/components/App.js [hunk 4]
```diff
@@ -8,26 +8,31 @@ import Form from './form/Form'
...
+      {appLoading && !appError && <Loading />}
+      {!appLoading && !appError &&
```
Comment (DrewBradley) [inline src/components/App.js:24]: Looks like code.

File: src/components/contacts/Contacts.css [hunk 5]
```diff
@@ -118,7 +118,7 @@
   width: 100%;
   height: 100%;
   background: black;
-  opacity: 50%;
+  opacity: 0.5;
```
Comment (DrewBradley) [inline src/components/contacts/Contacts.css:121]: Good catch!

File: src/context/queries.js [hunk 6]
```diff
@@ -85,6 +85,7 @@ export const ORGANIZATION_QUERY = gql`
 export const USER_QUERY = gql`
   query {
   user(id:1) {
+      id
```
Comment (DrewBradley) [inline src/context/queries.js:88]: This one too.

**Review by DrewBradley — APPROVED** [review]
So many small details to create such a large impact.

**Pull request merged**